gtsfm/multi_view_optimizer.py

```diff
@@ -77,9 +78,21 @@ def create_computation_graph(
 
         init_cameras_graph = dask.delayed(init_cameras)(wRi_graph, wti_graph, intrinsics_graph)
 
-        ba_input_graph = self.data_association_module.create_computation_graph(
+        data_assoc_graph = self.data_association_module.create_computation_graph(
             init_cameras_graph, v_corr_idxs_graph, keypoints_graph
         )
+        ba_input_graph = data_assoc_graph[0]
+        data_assoc_metrics_graph = data_assoc_graph[1]
+
+        auxiliary_graph_list = []
+        auxiliary_graph_list.append(
```

Comment:
nit: cant we create `auxiliary_graph_list` like this?
``` 
auxiliary_graph_list = [
 dask.delayed(io.save_json_file)(
                os.path.join("result_metrics", "data_association_metrics.json"), data_assoc_metrics_graph
            )
]
```

Reply:
nice, fixed that